src/containers.js

```diff
@@ -313,7 +271,7 @@ const ChatStackChild = new Lang.Class({
                                                   '',
                                                   GObject.ParamFlags.READWRITE |
                                                   GObject.ParamFlags.CONSTRUCT_ONLY,
-                                                  TriggerableEventQueue.$gtype),
+                                                  Queue.TriggerableEventQueue.$gtype),
```

Comment:
Should we rename then TriggerableEventQueue to TriggerableEvent as it is already under the Queue namespace?